### ISSUE TITLE
`prefer-minimal-ternary`: `checkComputedMemberAccess` should handle property reads, not just method calls

### ISSUE THREAD

**Issue by abrahamguo** [sindresorhus/eslint-plugin-unicorn#3669]
### Description

Just by name, `prefer-minimal-ternary` -> `checkComputedMemberAccess` sounds like it should allow the rule to convert to computed member access.

However, a detail buried in the docs clarifies that `checkComputedMemberAccess` only applies to method calls, not property reads.

Since this is already an off-by-default option, IMO, it makes sense that if a user wants to turn this on, they would expect — and be fine with — this style for both method calls and property reads.

### Examples

```js
// ❌
a ? b.c : b.d;

// ✅
b[a ? 'c' : 'd'];
```
